### PR TITLE
Changed a parameter name from VaultName to Name

### PR DESCRIPTION
**Pull request by Skuldo**
Changed a parameter name from VaultName to Name, and then added VaultName as an alias to make RemoveAzureKeyVault consistent with NewAzureKeyVault.

## Description

See https://github.com/Azure/azure-powershell/issues/10673

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by msJinLei** [comment]
@Skuldo Please add test cases & changelog to changelog.md

**Comment by msJinLei** [comment]
> Updated ChangeLog #10687
> 
> I don't think any tests need amending, this is essentially just re-ordering the alias/class name, the existing way of interacting with it is identical, with the addition of an extra alias.

Please merge the changelog commit into this PR.

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

**Comment by dingmeng-xue** [comment]
We should use VaultName as default parameter name and Name as its alias.

**Comment by msJinLei** [comment]
@Skuldo Sorry, one more thing, help mark down file should also be upgraded.

File: src/KeyVault/KeyVault/Commands/RemoveAzureKeyVault.cs [hunk 1]
```diff
@@ -54,7 +54,8 @@ public class RemoveAzureKeyVault : KeyVaultManagementCmdletBase
             HelpMessage = "Specifies the name of the key vault to remove.")]
         [ResourceNameCompleter("Microsoft.KeyVault/vaults", "ResourceGroupName")]
         [ValidateNotNullOrEmpty]
-        public string VaultName { get; set; }
+        [Alias("VaultName")]
+        public string Name { get; set; }
```
Comment (msJinLei) [inline src/KeyVault/KeyVault/Commands/RemoveAzureKeyVault.cs]: Is the original reference to VaultName required to be modified?

Comment (isra-fel) [reply src/KeyVault/KeyVault/Commands/RemoveAzureKeyVault.cs]: So @Skuldo , is there any particular reason why not keep `VaultName` as-is, and add `Name` as an alias?

**Comment by msJinLei** [comment]
Hi @Skuldo, we are going to close this pr since no update for a long time. You can submit a new one if the change is still needed.

**Pull request merged**